src/main/kotlin/no/nav/syfo/senoppfolging/v2/SenOppfolgingControllerV2.kt

```diff
@@ -105,14 +109,19 @@ class SenOppfolgingControllerV2(
         }
 
         val createdAt = LocalDateTime.now()
-        val latestVarsel = varselService.getUtsendtVarsel(personident)
+        val validUtsendtVarsel =
```

Comment:
Kanskje bare utsendtVarsel, siden det finnes vel ikke noen case for invalidVarsel